modules/managed-identity/user-assigned-identity/README.md

```diff
@@ -0,0 +1,68 @@
+| `name`            | `string` | No       | Optional. Name of the User Assigned Identity.                                                                                                                                                                                                                                                                                                                                                                  |
+| `location`        | `string` | No       | Optional. Location for all resources.                                                                                                                                                                                                                                                                                                                                                                          |
+| `lock`            | `string` | No       | Optional. Specify the type of lock.                                                                                                                                                                                                                                                                                                                                                                            |
+| `roleAssignments` | `array`  | No       | Optional. Array of role assignment objects that contain the 'roleDefinitionIdOrName' and 'principalId' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11' |
+| `tags`            | `object` | No       | Optional. Tags of the resource.                                                                                                                                                                                                                                                                                                                                                                                |
+
+## Outputs
+
+| Name              | Type   | Description                                                     |
+| :---------------- | :----: | :-------------------------------------------------------------- |
+| name              | string | The name of the user assigned identity                          |
+| resourceId        | string | The resource ID of the user assigned identity                   |
+| principalId       | string | The principal ID of the user assigned identity                  |
+| resourceGroupName | string | The resource group the user assigned identity was deployed into |
+
+## Examples
+
+### Example 1
+
+Deployment of `1` user assigned identity using the minimum required parameters.
```

Comment:
Why are the numbers code-fenced?

Reply:
To highlight it. Alternatively I could make them **bold**, but I personally prefer `code-fenced`.  Happy to undo it, if you rather prefer plain text

Reply:
done